code/typescript/main-workshop/lambda/hello.js

```diff
@@ -1,8 +1,8 @@
-exports.handler = async function(event) {
+exports.handler = async function (event) {
   console.log('request:', JSON.stringify(event, undefined, 2));
   return {
     statusCode: 200,
     headers: { 'Content-Type': 'text/plain' },
-    body: `Hello, CDK! You've hit "${event.path}". That's not very polite, "${event.path}" did not do anything to you...\n`
```

Comment:
Other workshops and the workshop guide for Typescript uses `Hello, CDK! You've hit "${event.path}".\n` so for sake of consistency I updated these lambdas to match.